modules/diffusion/mg_diffusion_solver.cc

```diff
@@ -709,14 +709,14 @@ MGDiffusionSolver::AssembleAbext()
     const auto fe_vol_data = cell_mapping.MakeVolumetricFiniteElementData();
     const size_t num_nodes = cell_mapping.GetNumNodes();
 
-    const auto& xs = matid_to_xs_map_.at(cell.material_id);
+    const auto& xs = matid_to_xs_map_.at(cell.block_id);
```

Comment:
there seems to be a `matid` here and I think this is perfectly fine as we are using a block id to pick a xs, which is a material property. so i like that we kept `matid` but if this was supposed to be changed and fell through the cracks, let's discuss

Reply:
I was not sure if I should change `matid_to_xs_map_`. I can see both `matid_to_xs_map_` and `block_id_to_xs_map_` as valid names, but `block_id_to_xs_map_` is probably a little bit more correct...

Reply:
block_id_to_xs_map_

Reply:
This should be done now.